dashboard/bower.json

```diff
@@ -33,10 +33,10 @@
     "zeroclipboard": "2.2.0",
     "angular-uuid4": "0.3.0",
     "angular-file-upload": "2.0.0",
-    "jquery": "2.1.3"
-  },
-  "devDependencies": {
+    "jquery": "2.1.3",
+    "js-yaml": "3.6.1"
```

Comment:
if you could add in PR comment that this new dependency is handled by https://dev.eclipse.org/ipzilla/show_bug.cgi?id=11918